### PR TITLE
Update ko_KR.lang

### PR DESCRIPTION
**Pull request by nark31**
Lang File Update.
Retranslate.

* Tallow block desc is not working
* [An Old Relic from the land of Oneteufyv] this mean is known.

### CODE REVIEW

**Comment by nark31** [comment]
Update Nation, Shingles Arcane, Diorite, Andesite, Granite, VoidStone, ArcaneStone, Futura, LargeHexPlating, Technical Block, Factoryblock2, StoneBrick

File: src/main/resources/assets/chisel/lang/ko_KR.lang [hunk 1]
```diff
@@ -1,538 +1,889 @@
...
+item.chisel.chisel.desc.gui=우클릭으로 GUI 열기
+item.chisel.chisel.desc.lc1=왼클릭으로 블록을 순차적으로 모양을 변경
+item.chisel.chisel.desc.lc2=끌에 변경된 블록이 있으면 해당 블록으로만 변경
+item.chisel.chisel.desc.modes=변환 범위를 선택 할 수 있습니다.
+
+item.chisel.chisel_diamond.name=다이아몬드 끌
+item.chisel.chisel_obsidian.name=흑요석 끌
 
-# Item localizations
-item.chisel.name=정
```
Comment (CptSpaceToaster) [inline src/main/resources/assets/chisel/lang/ko_KR.lang]: You may want this

File: src/main/resources/assets/chisel/lang/ko_KR.lang [hunk 2]
```diff
@@ -1,538 +1,889 @@
...
-tile.marble.9.desc=둥근 조각의 대리석
-tile.marble.10.desc=파인 대리석
-tile.marble.11.desc=크게 파인 대리석
-tile.marble.12.desc=대리석 타일
-tile.marble.13.desc=Arranged Marble Tiles
-tile.marble.14.desc=화려한 대리석 타일
-tile.marble.15.desc=대리석 블록
-
-#Marble Slabs
-tile.chisel.marbleSlab.name=대리석 반블럭
```
Comment (CptSpaceToaster) [inline src/main/resources/assets/chisel/lang/ko_KR.lang]: You changed the localization a bunch of times like this.
marbleSlab->marble_slab

Was this intentional?  You did it several times

Comment (nark31) [inline src/main/resources/assets/chisel/lang/ko_KR.lang]: translate, based on the en_US.lang file.
Refer. http://puu.sh/h7aIt/350ab0a505.png

**Comment by CptSpaceToaster** [comment]
looks good then.

**Comment by nark31** [comment]
thanks! :>

**Pull request merged**